hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationSupervisor.java

```diff
@@ -438,20 +467,49 @@ public long getMaxReplicationStreams() {
     }
   }
 
+  private long getCount(Map<String, AtomicLong> counter) {
+    if (counter.isEmpty()) {
+      return 0;
+    }
+    AtomicLong total = new AtomicLong(0);
```

Comment:
This does not need to be an AtomicLong - a simple Long is fine. This is a local variable and we return only its value, so it does not need to be thread safe.

Not particularly important, but you could also remove the `if (counter.isEmpty())` section:

```
long counter = 0;
counter.forEach((key, value) -> total += value.get();
return counter;
```

Reply:
Thanks @sodonnel  for your comment and review.
I have updated it, can you review it again?
Thanks.